test/testDominatingSet.cpp

```diff
@@ -0,0 +1,185 @@
+// #include <iostream>
```

Comment:
Please rewrite tests so that they are similar in spirit to e.g. testVertexColoring

In general, test is for small/corner cases, benchmark is for large suites (e.g. big g6 input files)